content/docs/get-started/index.md

```diff
@@ -158,6 +158,8 @@ Try creating and running this simple `predict.py` script:
 ```py
 from mlem.api import load
 
+import pandas as pd
+
```

Comment:
Thanks @thecaffeinedev! But we skipped that for the sake of brevity :) So let's remove this

```suggestion
```

Reply:
@aguschin Thanks for the response. I have removed the Get started changes now.